src/canisters/frontend/ic-asset/src/asset_config.rs

```diff
@@ -46,6 +47,7 @@ pub struct AssetConfigRule {
     headers: Maybe<HeadersConfig>,
     #[serde(skip_serializing_if = "Option::is_none")]
     ignore: Option<bool>,
+    enable_aliasing: Option<bool>,
```

Comment:
This keeps the output from getting cluttered with defaults.  It might even make some of the changes to e2e tests unnecessary.
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    enable_aliasing: Option<bool>,
```